Source/Models/EThreeParams.swift

```diff
@@ -60,6 +60,8 @@ import VirgilCrypto
     @objc public var serviceUrls: ServiceUrls
     /// Initial Jwt
     @objc public var initialJwt: Jwt? = nil
+    /// Security application group ID, if defined then resources will be located in the container directory associated with it
+    @objc public var securityApplicationGroupIdentifier: String? = nil
```

Comment:
Maybe we should call it smaller, e.g. `appGroup`? This is the way apple calls it in Capabilities, so should be recognizable enough.

Reply:
Maybe.
The point behind my decision is how Apple picks names for its API, i.e
```swift
func containerURL(forSecurityApplicationGroupIdentifier: String) -> URL?
```
https://developer.apple.com/documentation/foundation/filemanager/1412643-containerurl